File: foundation/foundation 2 month/lesson1/homework/ex_4.c
#include <stdio.h>
void check(int n)
{
	if(n % 2 == 0)
{
	puts("Juft");
}
	else
	puts("Toq");
}
	int main()
{
	int number;
	printf("Sonni kiriting: ");
	scanf("%d",&number);
	check(number);
	return 0;
}

File: full stack/lesson2/homework/ex_4.js
let number = 141
let number_1 = number
let check = 0
let j = String(number).length
while(j>0)
{
    let a = number % 10
    check = check * 10 + a
    number = Math.floor(number / 10)
    j-=1
}
if (number_1 == check)
{
    console.log(true)
}
else
{
    console.log(false)
}